diff --git a/backend/src/main/java/org/lvtn/mws/infrastructure/persistence/entity/ProductEntity.java b/backend/src/main/java/org/lvtn/mws/infrastructure/persistence/entity/ProductEntity.java
index 5f89c51..6eeaefb 100644
--- a/backend/src/main/java/org/lvtn/mws/infrastructure/persistence/entity/ProductEntity.java
+++ b/backend/src/main/java/org/lvtn/mws/infrastructure/persistence/entity/ProductEntity.java
@@ -46,7 +46,7 @@ public class ProductEntity {
     @Column(name = "weight", precision = 10, scale = 2)
     private BigDecimal weight;
 
-    @Column(name = "volume", precision = 10, scale = 2)
+    @Column(name = "volume", precision = 12, scale = 6)
     private BigDecimal volume;
 
     @Column(name = "hazardous_flag")
